autopart-hibernation.ks.in

```diff
@@ -0,0 +1,70 @@
+#test name: autopart-hibernation
+%ksappend repos/default.ks
+network --bootproto=dhcp
+
+zerombr
+clearpart --all --initlabel
+autopart --hibernation
+
+%ksappend common/common_no_storage_and_payload.ks
+
+%packages
+%end
+
+%pre
+%end
```

Comment:
Empty `%pre` can be removed now.

Reply:
What about %packages? Should I keep it there?

Reply:
Right, I think it should be replaced by an existing fragment: `%ksappend payload/default_packages.ks`